Firefox_Private_Network_ToS/en-US.md

```diff
@@ -0,0 +1,82 @@
+# Firefox Private Network Browser Extension (Beta) Terms of Service
+
+Version 3.0, effective July 7, 2020
```

Comment:
Use the same page title and date format like the one you did: https://github.com/mozilla/legal-docs/pull/1463/files

Reply:
We need to be more consistent about this. Let's keep effective. I'll update the others to match this standard.